src/frontend/Ast_to_Mir.ml

```diff
@@ -1,44 +1,54 @@
 open Core_kernel
+open Common
```

Comment:
I would suggest opening all of the relevant MIR types up top again - like UnsizedType etc. I'm not a big fan of fully qualified access being required at all call-sites.

Reply:
It is considered best practice to open modules as close to call site as possible. Further, since all modules have a type `t` this would mean you would still need to qualify the types.